projects/novo-elements/src/elements/form/Control.ts

```diff
@@ -265,11 +265,13 @@ export class NovoControlElement extends OutsideClick implements OnInit, OnDestro
 
   get showCount() {
     let charCount: boolean =
-      (this.form.controls[this.control.key].maxlength &&
-        this.focused &&
-        (this.form.controls[this.control.key].controlType === 'text-area' ||
-          this.form.controls[this.control.key].controlType === 'textbox')) ||
-      (this.form.controls[this.control.key].maxlength && this.form.controls[this.control.key].controlType === 'picker');
+      this.focused &&
+      this.form.controls[this.control.key].maxlength &&
+      (
+        this.form.controls[this.control.key].controlType === 'text-area' ||
```

Comment:
Loving this boolean simplification. I think it might be clearer if it were an `Array.prototype.includes` check for those values instead of accessing controlType for each check i.e. `['textbox', 'picker', 'text-area'].includes(controlType)`

Reply:
Got that array in a const for readability now.